lms/djangoapps/commerce/tests/test_views.py

```diff
@@ -1,18 +1,6 @@
 """ Tests for commerce views. """
```

Comment:
Is there a reason why this entire file wasn't removed? Is something using that mixin?

Reply:
It's still used in https://github.com/edx/edx-platform/blob/master/lms/djangoapps/commerce/api/v0/tests/test_views.py#L31 and https://github.com/edx/edx-platform/blob/master/lms/djangoapps/commerce/api/v1/tests/test_views.py#L25 (and those views are still used in production).  We could move it, but those two directories don't share a common test dir ancestor.  I guess `lms/djangoapps/commerce/tests/__init__.py` would be marginally better, but I was opting for minimal changes.